x-pack/plugins/security_solution/public/resolver/view/use_camera.ts

```diff
@@ -36,7 +43,10 @@ export function useCamera(): {
    * to determine where it belongs on the screen.
    * The projection matrix changes over time if the camera is currently animating.
    */
-  const projectionMatrixAtTime = useSelector(selectors.projectionMatrix);
+
+  const projectionMatrixAtTime = useSelector((state: State) => {
```

Comment:
I suspect the issue might be here. The `useSelector` calls used to be passed stable references to functions. This allows them to be cached. Without being cached, `projectionMatrixAtTime` will always return a new value. This will invalidate the useEffect that calls it below:

```
  /**
   * Keep the projection matrix state in sync with the selector.
   * This isn't needed during animation.
   */
  useLayoutEffect(() => {
    // Update the projection matrix that we return, rerendering any component that uses this.
    setProjectionMatrix(projectionMatrixAtTime(sideEffectors.timestamp()));
  }, [projectionMatrixAtTime, sideEffectors]);
```

Try using useCallback around all the function expressions you pass to `useSelector` when they were previously references to functions. E.g.

```
const projectionMatrixAtTime = useSelector(useCallback((state: State) => {
 return selectors.projectionMatrix(state.analyzer.analyzerById[id]);
},[]));
```